backend/accounts.go

```diff
@@ -63,58 +64,92 @@ func (b *backend) listAccounts(ctx context.Context, req *logical.Request, data *
 }
 
 func (b *backend) createAccount(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
-  privateKey, _ := crypto.GenerateKey()
-  defer ZeroKey(privateKey)
-  privateKeyBytes := crypto.FromECDSA(privateKey)
-  privateKeyString := hexutil.Encode(privateKeyBytes)[2:]
-
-  publicKey := privateKey.Public()
-  publicKeyECDSA, _ := publicKey.(*ecdsa.PublicKey)
-  publicKeyBytes := crypto.FromECDSAPub(publicKeyECDSA)
-  publicKeyString := hexutil.Encode(publicKeyBytes)[4:]
-
-  hash := sha3.NewLegacyKeccak256()
-  hash.Write(publicKeyBytes[1:])
-  address := hexutil.Encode(hash.Sum(nil)[12:])
-
-  accountPath := fmt.Sprintf("accounts/%s", address)
-
-  accountJSON := &Account{
-    Address:      address,
-    PrivateKey:   privateKeyString,
-    PublicKey:    publicKeyString,
-  }
-
-  entry, _ := logical.StorageEntryJSON(accountPath, accountJSON)
-  err := req.Storage.Put(ctx, entry)
-  if err != nil {
+	keyInput := data.Get("privateKey").(string)
```

Comment:
If we feel it's a common way to specify private key, we could strip out `0x` from the input if present, instead of requiring the user to not include it.